pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -445,7 +445,8 @@ def DecimalArray__my_sum(self):
     result = df.groupby("id")["decimals"].agg(lambda x: x.values.my_sum())
     tm.assert_series_equal(result, expected, check_names=False)
     s = pd.Series(DecimalArray(data))
-    result = s.groupby(np.array([0, 0, 0, 1, 1])).agg(lambda x: x.values.my_sum())
+    grouper = np.array([0, 0, 0, 1, 1], dtype=np.int64)
```

Comment:
i think i asked elsewhere: specifying this is benign, just affects result.index.dtype or something like that?